_posts/2015-09-17-modern-blog.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title:  "ModernBlog"
+date:   2015-0917 01:01:00
+homepage: https://github.com/inded/Jekyll_modern-blog
+download: https://github.com/inded/Jekyll_modern-blog/archive/gh-pages.zip
+demo: http://inded.github.io/Jekyll_modern-blog/
+author: Wferr
+thumbnail: modern-blog.png
```

Comment:
Since the theme name has no space in the title, then thumbnail filename should be `modernblog.png`.